Octokit/Models/Request/MilestoneRequest.cs

```diff
@@ -39,8 +39,10 @@ internal string DebuggerDisplay
 
     public enum MilestoneSort
     {
-        [Parameter(Value = "due_date")]
+        [Parameter(Value = "due_on")]
```

Comment:
Ive pushed up changes to set this back to "due_date" (and also contacted GitHub support to ask them about the doc/api correctness for this call)